test/unittests/eof_validation_test.cpp

```diff
@@ -562,44 +558,44 @@ TEST_F(eof_validation, EOF1_rjumpi_invalid_destination)
 TEST_F(eof_validation, EOF1_rjumpv_invalid_destination)
 {
     // table = [-23] case = 0
-    add_test_case("EF0001 010004 0200010008 040000 00 00800000 6000E200FFE96001",
-        EOFValidationError::invalid_rjump_destination);
+    add_test_case(
+        eof_bytecode(rjumpv({-23}, 0) + push(1), 1), EOFValidationError::invalid_rjump_destination);
```

Comment:
It would make sense to add terminating OP_STOP in all cases in this test, to make them invalid only for one reason.